src/wizards/tokenSwapDealWizard/stages/termsStage/termClause/termClause.html

```diff
@@ -32,8 +32,13 @@ <h2 if.bind="clause.title" class="clause__title">${clause.title}</h2>
         max-length="125"
         show-counter
       >
-        <pinput-text view-model.ref="clauseTitleInputViewModel" max="125" value.bind="clause.title & validate"></pinput-text>
+        <pinput-text
```

Comment:
could you please check your editor settings, that you have the following setting

We are `preserve`ing html attribute wrap
![image](https://user-images.githubusercontent.com/30693990/186683998-c2342ab4-c6fd-48f8-a187-3fde1630a2be.png)

Reply:
done, thanks